xml/admin_ceph_gateway.xml

```diff
@@ -2906,68 +2932,83 @@ rgw_override_bucket_index_max_shards = 12
       "metadata_heap": "",
       "realm_id": ""
   }</screen>
-</sect3>
-  <sect3 xml:id="ceph-rgw-fed-deldefzonegrp">
-   <title>Deleting the Default Zone and Group</title>
-   <para>
-    The default installation of &ogw; creates the default &zgroup; called
-    <literal>default</literal>. Delete the default zone if it exists.
-    Make sure to remove it from the default &zgroup; first.
-   </para>
-   <important>
-     <para>The following steps assume a multi-site configuration using newly
-       installed systems that are not storing data yet. <emphasis role="bold">Do not delete</emphasis>
-       the default zone and its pools if you are already using it to store data,
-       or the data will be deleted and unrecoverable.</para>
-   </important>
+   </sect3>
+   <sect3 xml:id="ceph-rgw-fed-deldefzonegrp">
+    <title>Deleting the Default Zone and Group</title>
+    <para>
+     The default installation of &ogw; creates the default &zgroup; called
+     <literal>default</literal>. Delete the default zone if it exists. Make
+     sure to remove it from the default &zgroup; first.
+    </para>
+    <important>
+     <para>
+      The following steps assume a multi-site configuration using newly
+      installed systems that are not storing data yet. <emphasis role="bold">Do
+      not delete</emphasis> the default zone and its pools if you are already
+      using it to store data, or the data will be deleted and unrecoverable.
+     </para>
+    </important>
 <screen>&prompt.cephuser;radosgw-admin zonegroup delete --rgw-zonegroup=default</screen>
-   <para>Delete the default pools in your &ceph; storage cluster if they exist:</para>
-   <important>
-     <para>The following step assumes a multi-site configuration using newly
-       installed systems that are not currently storing data. <emphasis role="bold">Do not delete</emphasis>
-       the default &zgroup; if you are already using it to store data.</para>
-   </important>
```

Comment:
I suggest moving the "important" warning _before_ the introductory text saying "delete the default pools..."